website_sale_wishlist_keep/static/src/js/tour.js

```diff
@@ -4,12 +4,13 @@
 odoo.define("website_sale_wishlist_keep.tour", function (require) {
     "use strict";
 
-    const tour = require("web_tour.tour");
-    const base = require("web_editor.base");
+    var tour = require("web_tour.tour");
 
-    const steps = [
+    var steps = [
```

Comment:
why?

Reply:
A test I had forgotten to reinstate but it wasn't the access problem that one, thanks!